index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project</title>
+    <link    rel="stylesheet"    href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"    />    <link    rel="stylesheet"    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"    /><!-- font awesome icons -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@100;400&family=Lato:wght@300;700;900&family=Poppins:wght@100;200;700&family=Roboto:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav class="nav-element">
+            <div class="nav-items">
+                <ul class="social-icons">
+                    <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+                    <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+                    <li><a href="">English</a></li>
+                    <li><a href="">My Page</a></li>
+                    <li><a href="#">Logout</a></li>
+                </ul>
+                <div class="nav-bar">
```

Comment:
- [ ] Your menu bars are almost there! Let's make the requested changes from the previous reviewer:

| Template | Your design |
|---|---|
|  ![Captura de pantalla 2022-04-15 a la(s) 10 22 18 p  m](https://user-images.githubusercontent.com/83717285/163659662-4d28cb98-62d2-45f2-a3c5-6f53dc6f88af.png) | ![Captura de pantalla 2022-04-15 a la(s) 10 11 55 p  m](https://user-images.githubusercontent.com/83717285/163659485-6c3a883a-d82d-4905-b6e8-e463d79a47d3.png)|

- Please add a border around the last menu item and use the font from the template. This should be done on both pages (index and about).